src/style/Header.module.css

```diff
@@ -0,0 +1,84 @@
+@import url('https://fonts.googleapis.com/css2?family=Roboto+Slab:wght@300;400;500;600;700;800&display=swap');
+
+.lesson_panel {
+  border: 1px solid #e8e8e8;
+  background: #fff;
+  padding: 2rem 7.188rem 1.625rem 1.688rem;
+  margin: 2.313rem 6.25rem 0;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+}
+
+.row_1 {
+  /* border-right: 1px solid #e8e8e8; */
```

Comment:
- [ ] [OPTIONAL] Same here kindly make sure there is no commented code in your codebase